cmd/debugger/main.go

```diff
@@ -297,17 +324,17 @@ func main() {
 
 		err := os.Setenv("TERM", debuggerSettings.Term)
 		if err != nil {
-			conslogger.Warnf("Failed to set term: %v", err)
+			conslogger.VerboseWarnf("Failed to set term: %v\n", err)
```

Comment:
similar to my comment below (apparently I started reviewing from the bottom up)...

> Was this warning commonly occurring? I would think having Setenv fail should be extremely rare, and should always show up as a warning -- if the TERM can't be set, feature like resizing the terminal won't work.

Reply:
it's almost as if some of this code was copied and pasted in a previous commit :see_no_evil: